kernel/src/table_properties/deserialize.rs

```diff
@@ -76,6 +76,15 @@ fn try_parse(props: &mut TableProperties, k: &str, v: &str) -> Option<()> {
         }
         "delta.checkpointPolicy" => props.checkpoint_policy = CheckpointPolicy::try_from(v).ok(),
         "delta.enableRowTracking" => props.enable_row_tracking = Some(parse_bool(v)?),
+        "delta.enableInCommitTimestamps" => {
+            props.enable_in_commit_timestamps = Some(parse_bool(v)?)
+        }
+        "delta.inCommitTimestampEnablementVersion" => {
+            props.in_commit_timestamp_enablement_version = Some(parse_int(v)?)
```

Comment:
Something went wrong:
```
error[E0425]: cannot find function `parse_int` in this scope
  --> /Users/runner/work/delta-kernel-rs/delta-kernel-rs/kernel/src/table_properties/deserialize.rs:83:65
   |
83 |             props.in_commit_timestamp_enablement_version = Some(parse_int(v)?)
   |                                                                 ^^^^^^^^^ not found in this scope

error[E0425]: cannot find function `parse_int` in this scope
  --> /Users/runner/work/delta-kernel-rs/delta-kernel-rs/kernel/src/table_properties/deserialize.rs:86:67
   |
86 |             props.in_commit_timestamp_enablement_timestamp = Some(parse_int(v)?)
   |                                                                   ^^^^^^^^^ not found in this scope
```
(I think all the other int properties use `parse_positive_int` and `parse_int` may not even exist?)

(these two do need to tolerate 0)

Reply:
yea sorry opened this as a sort of "todo" and will fix this :)

Reply:
sorry should have probably made it a draft...